README.md

```diff
@@ -88,14 +88,14 @@ for additional information):
 
     $ sudo apt-get install liboce-foundation-dev liboce-modeling-dev liboce-ocaf-dev liboce-visualization-dev liboce-ocaf-lite-dev
 
-If not, you will need to compile Open CASCADE yourself:
+If not, you will need to compile Open CASCADE yourself (note that the build takes a long time):
 
     $ sudo apt-get install libftgl-dev libtbb2 libtbb-dev libgl1-mesa-dev libfreetype6-dev
     $ git clone https://github.com/tpaviot/oce.git
     $ cd oce
     $ mkdir build && cd build
     $ cmake ..
-    $ make
+    $ make -j
```

Comment:
Is this a good suggestion, wouldn't it spawn too many processes when using `-j` without an integer? I have no experience with this.